### ISSUE TITLE
RN 0.71 Android build still failing for `@segment/analytics-react-native-advertising-id`

### ISSUE THREAD

**Issue by roryf** [segmentio/analytics-react-native#796]
- `analytics-react-native` version: 2.13.4
- Integrations versions (if used):
  - @segment/analytics-react-native-plugin-advertising-id: 1.2.2
- React Native version: 0.71.4
- iOS or Android or both? Android

**Steps to reproduce**
- Install RN 0.71
- `npm run android`

**Expected behavior**

It builds successfully

**Actual behavior**

Fails same as in #770

**Comment by roryf** [comment]
The following with `patch-package` has worked around this for us currently:

```
diff --git a/node_modules/@segment/analytics-react-native-plugin-advertising-id/android/gradle.properties b/node_modules/@segment/analytics-react-native-plugin-advertising-id/android/gradle.properties
index c3264f9..59eda98 100644
--- a/node_modules/@segment/analytics-react-native-plugin-advertising-id/android/gradle.properties
+++ b/node_modules/@segment/analytics-react-native-plugin-advertising-id/android/gradle.properties
@@ -1,3 +1,3 @@
-AnalyticsReactNativePluginAdvertisingId_kotlinVersion=1.6.0
+AnalyticsReactNativePluginAdvertisingId_kotlinVersion=1.7.21
 AnalyticsReactNativePluginAdvertisingId_compileSdkVersion=31
 AnalyticsReactNativePluginAdvertisingId_targetSdkVersion=31

```

**Comment by alanjcharles** [comment]
thanks for letting us know @roryf fix in #799

**Comment by alanjcharles** [comment]
fixed in version [1.2.3](https://npmjs.com/package/@segment/analytics-react-native-plugin-advertising-id)
